src/index.html

```diff
@@ -1,13 +1,64 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="content">
+      <div class="stars--0">
+        <div class="star item"></div>
+        <div class="star item"></div>
+        <div class="star item"></div>
+        <div class="star item"></div>
+        <div class="star item"></div>
+        <div class="star item"></div>
+      </div>
```

Comment:
![Screenshot from 2023-04-02 15-25-00](https://user-images.githubusercontent.com/33029864/229352804-c1803062-3708-41bb-881a-2b255f617df9.png)
should be five stars in row
stars wil be responsible for general styles of row, stars--0 stars--1...stars--5 modifiers wil be responsible only for amount of active stars
```suggestion
      <div class="stars stars--0">
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars __star"></div>
      </div>
```